flash/website/models.py

```diff
@@ -2,18 +2,28 @@
 from django.contrib.auth.models import User
 import random
 
-NUM_BOXES = 3
-BOXES = range(1, NUM_BOXES + 1)
+
+class Box(models.Model):
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    box_number = models.IntegerField()
+
+    @staticmethod
+    def get_unique_boxes(user):
+        boxes_with_cards = Box.objects.filter(user=user).values('box_number').distinct()
```

Comment:
I think user_boxes would be a better name

Reply:
Done!